kensu/utils/reporters.py

```diff
@@ -105,6 +108,16 @@ def apply(self, obj, kensu_api, method):
         return obj
 
 
+class DictReporter(Reporter):
+    def __init__(self, config, level=None):
+        super().__init__(config)
+        self.entities = defaultdict(list)
+
+    def apply(self, obj, kensu_api, method):
+        self.entities[obj.__class__].append(obj)
```

Comment:
would it be more human friendly to use short string as a key? e.g. `User`, `DataSource`.